packages/game/package.json

```diff
@@ -1,16 +1,24 @@
 {
   "name": "@adventure-bot/game",
-  "version": "1.0.0",
+  "version": "0.0.0",
   "description": "Slay a monster! Stab your friends! Adventure bot!",
   "main": ".build/index.js",
+  "types": ".build/index.d.ts",
+  "files": [
+    ".build",
+    "src",
+    "tsconfig.json"
+  ],
   "scripts": {
+    "start": "echo 'Starting Adventure Bot from the CLI has been moved to `yarn game serve`'",
+    "serve": "ts-node -r tsconfig-paths/register bin/adventure-bot.ts",
+    "dev": "nodemon --config nodemon-dev.json",
+    "build": "yarn generate-asset-manifest && yarn build:ts",
+    "build:ts": "ttsc",
```

Comment:
Separated a `build` and `build:ts` so we don't needlessly keep generating the asset manifest. Will optimize soon.